### PR TITLE
Css canvas medical-Dylan De Vei

### PR DESCRIPTION
**Pull request by dylandevei**
https://user-images.githubusercontent.com/99229316/159205481-1ed56659-d16a-4f3a-94bb-b57da54b4daf.mov

Screen Shot 2022-03-20 at 9 26 03 PM
Screen Shot 2022-03-20 at 9 26 11 PM

### CODE REVIEW

File: css-canvas-medical/index.html [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <div class="row">
+      <div class="column-fourth align-items-center justify-content-flex-start padding-l-50">
+        <img src="images/logo-blue.svg" alt="blue logo" class="blue-logo-image-size">
+      </div>
+      <div class="column-fourth"></div>
```
Comment (codingcodymiller) [inline css-canvas-medical/index.html]: You should never have empty `column` elements. Instead, you should make the previous `column-fourth` into a `column-half`

File: css-canvas-medical/index.html [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+    </div>
+
+    <div class="row background-white padding-bottom-25">
+      <div class="column-one-third space-evenly align-items-center justify-content-flex-start padding-l-50">
+        <section>
+          <h3 class="tiempos-headline grey-text">Ready to learn more?</h3>
+          <p class="avenir-500 grey-text padding-top-5 font-14">Schedule a private meeting now</p>
+        </section>
+      </div>
+      <div class="column-one-third"></div>
```
Comment (codingcodymiller) [inline css-canvas-medical/index.html]: Same deal here, no empty `column` elements

File: css-canvas-medical/index.html [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+      <div class="column-fourth padding-l-50">
+        <img src="images/logo-white.png" alt="white-logo" class="blue-logo-image-size image-margin">
+        <section class="white-text line-height-smaller">
+          <p>2307 Irving Street, Suite 228</p>
+          <p>San Fransisco CA 94122</p>
+          <p>(800) 370-1416</p>
+          <p>info@canvasmedical.com</p>
+        </section>
+      </div>
+      <div class="column-fourth"></div>
```
Comment (codingcodymiller) [inline css-canvas-medical/index.html]: Same deal here, no empty `column` elements

File: css-canvas-medical/index.html [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+      </div>
+
+    </div>
+    <div class="row background-grey">
+      <div class="column-one-third space-evenly grey-text padding-tb-20">
+        <p class="font-14">Privacy &amp; Terms</p>
+        <p class="font-14">Contact Us</p>
+      </div>
+      <div class="column-one-third"></div>
+      <div class="column-one-third"></div>
```
Comment (codingcodymiller) [inline css-canvas-medical/index.html]: Same deal here, no empty `column` elements

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great work!! A few notes for you:

**Comment by dylandevei** [comment]
https://user-images.githubusercontent.com/99229316/159400505-f27da00d-95f0-4476-8368-2faa9f7fe74f.mov

revisions to canvas medical, I took out the empty column elements and adjusted some minor things.

**Review by codingcodymiller — APPROVED** [review]
Great work!! 🎉

**Pull request merged**